router/Cargo.toml

```diff
@@ -10,6 +9,6 @@ documentation = "http://ironframework.io/doc/router/index.html"
 keywords = ["iron", "web", "http", "routing", "router"]
 
 [dependencies]
-route-recognizer = "0.1"
+route-recognizer = "0.1.13"
```

Comment:
There should be no need for specifying a patch version, `cargo update` should be enough.